### ISSUE TITLE
Exception ticking World Component: RimWorld.Planet.TileTemperaturesCompSystem.Threading.ThreadAbortException

### ISSUE THREAD

**Issue by UndeadKitten** [cseelhoff/RimThreaded#411]
**Describe the bug**
After I had defeated a mechanoid raid, my fps dropped to less than 1 fps. Using the same save game as my previous posts. I only have Harmony and RimThreaded enabled, and I don't have Royalty DLC. I am subscribed to other mods this time but none of them are enabled and I have never used any other mods on this save. Restarting Rimworld fixed the issue.
EDIT: Version was RT 1.2.3

**Error Log**
https://gist.github.com/UndeadKitten/4acf5ff84fcbd626f9bde2cc7c3ed905

**Specs**
Operating System: Windows 10 Home 64-bit
Processor: Intel(R) Core(TM) i5-4690K CPU @ 3.50GHz (4 CPUs), ~3.5GHz
Memory: 16384MB RAM
Card name: NVIDIA GeForce GTX 970

**Comment by pastorismylord** [comment]
can you please try t recreate this error with rimthreaded 1.2.4?

**Comment by pastorismylord** [comment]
any advice on how to reproduce this error? do you have a save available for this error?

**Comment by UndeadKitten** [comment]
I have continued to play the same save and I have not encountered this specific error again since posting this. I disabled dev-mode for a portion of my playtime as it gets annoying with the log regularly popping up. I have encountered times where my FPS and TPS dropped considerably (720 TPS down to 120-360 TPS), which was fixed by restarting my game, or letting it play out for a while then restarting my game. I didn't check what the error was during those times though.

(Below is copy and pasted across a few of my posts)

I don't know of a way to reliably reproduce any of the errors that I get, they happen seemingly at random while playing. Sometimes almost immediately upon loading up the save and sometimes hours later. I get many different errors, more than what I have posted here. I regularly get inconsequential errors and sometimes ones that break functions of the game (usually pawn unable to reach area) that are fixed upon restarting the game. Both on 1.2.3 and 1.2.4 preview.

If you have any suggestions for me on what to do then let me know.
